src/pages/getting-started/helpdesk-apps/helpdesk-api/index.mdx

```diff
@@ -9,4 +9,12 @@ desc: "Learn about HelpDesk API and where they are located in our documentation.
 
 # HelpDesk API
 
-To read about public API for HelpDesk, go to the <a href="https://api.helpdesk.com/docs" target="_blank">HelpDesk API Documentation</a>.
+<a href="https://api.helpdesk.com/docs" target="_blank">HelpDesk API</a> is necessary when you build or improve your HelpDesk applications. That is why it's fully accessible to experiment with and discover its broad potential. The resources that can be managed using HelpDesk API are:
+
```

Comment:
I believe there is some kind of a bug or something like that here, as it is impossible to make space visible on the preview here. however on the other side, when I put the listed things in one line and just listed them after comas, then it looks much worse that this option without a space between lines.